src/qwed_new/api/verification_context_routes.py

```
@@ -0,0 +1,158 @@
+        session.add(
+            VerificationLog(
+                organization_id=tenant.organization_id,
+                user_id=getattr(tenant, "user_id", None),
+                query=query,
+                result=json.dumps(result),
+                is_verified=is_verified,
+                domain="VERIFICATION_CONTEXT",
+            )
+        )
+        session.commit()
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)